RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -16,8 +16,22 @@
 #include "RecoTracker/MkFitCore/standalone/Event.h"
 #endif
 
+#ifndef MKFIT_STANDALONE
+#include "FWCore/Utilities/interface/isFinite.h"
+#endif
+
 #include <algorithm>
 
+namespace {
+  constexpr bool isFinite(float x) {
```

Comment:
According to cppreference `std::isfinite()` is not `constexpr`
https://en.cppreference.com/w/cpp/numeric/math/isfinite
Possibly a gcc extension?

Reply:
OK. I dropped constexpr.
I did compile with `constexpr ...{return std::isfinite` here, in standalone, and even with clang before committing.

I think I have seen/done something like this before and recall concluding that it was perhaps inline-related (as if, if everything is implemented inline the compiler does not pay attention to explicit absence of constexpr)

Reply:
Could be. We've also seen `libstdc++` to implement some functions as `constexpr` that are not (yet?) in the standard. Looking into `cmath` header shipped with GCC 9.3.0, `isfinite` appears to be `constexpr` there unless `__CORRECT_ISO_CPP11_MATH_H_PROTO_FP` is defined (I'd _guess_ we don't define that?).